samples/amber.cc

```diff
@@ -451,14 +456,16 @@ int main(int argc, const char** argv) {
       // give clues as to the failure.
     }
 
-    if (!options.image_filename.empty()) {
+    auto fb_name = options.fb_names.begin();
+
+    for (const auto& image_filename : options.image_filenames) {
```

Comment:
I think I'd prefer turning this into:

```
for (size_t i = 0; i < options.image_filenames.size(); ++i)
```

and then removign fb_name above and using options.fb_names[i] below.